states/HI/script.js

```diff
@@ -0,0 +1,52 @@
+//
+// ** Special handling for Hawaii governor. **
+// Hawaii has a 'region' dropdown that doesn't correlate directly to any address component we currently collect.  So we have connected the region to a county where we can, and a zip code where we cannot.  See comments in this file for more details.
+
+// This JS is included in a separate file for readability and ease of editing.  All comments have been removed, and copied into the yaml.  The same steps should take place should this JS need to be edited.
```

Comment:
@jason44406 Hm is it possible to include comments in the script in the yaml file?

Reply:
I had trouble getting the execute_script to work when there were comments inside of the script value in the yaml.  I could put a summary inside of the yaml and ditch the script.js if that is preferable.  I don't care either way -- I just wanted to do whatever was easiest for documentation and future edits.

Reply:
I wonder if they need to be escaped or something, which will likely be incompatible in the original js source.  I'm fine with the separate file, but I would make a very clear comment in the YAML indicating that one _should NOT_ edit the js in the YAML.

Reply:
I found no good way to include comments inside the JS string.  I instead summarized the comments in the YAML and axed the script.js.